README.md

```diff
@@ -1,2 +1,93 @@
 # o2r-guestlister
-An OAuth server implementation to allow offline login with o2r-bouncer 
+
+A Node.js OAuth2 server implementation to allow offline login with o2r-bouncer as part of the [o2r reference-implementation](https://github.com/o2r-project/reference-implementation).
+
+Based on the repository https://github.com/gerges-beshay/oauth2orize-examples and the underlying OAuth2 server implementation [oauth2orize](https://www.npmjs.com/package/oauth2orize).
+
+
+Requirements:
+
+- nodejs `>= 6.2`
+- npm
+
+## Endpoints
```

Comment:
Please add a link to a reasonable OAuth2 documentation page, which help you understand the process.

Reply:
done